neuraxle_tensorflow/tensorflow_v2.py

```diff
@@ -184,7 +172,12 @@ def save_step(self, step: 'Tensorflow2ModelStep', context: 'ExecutionContext') -
         :type context: ExecutionContext
         :return: saved step
         """
-        step.checkpoint_manager.save()
+        checkpoint_manager = tf.train.CheckpointManager(
+            step.checkpoint,
+            context.get_path(),
```

Comment:
Maybe `os.path.join(context.get_path(), summary_id)` ? 

Otherwise, how do we differentiate between different runs with different hyperparams when saving and reloading ? It should be simple to be compatible with all the rest of the mechanics of Neuraxle here.

Reply:
However, I recognize that from one minibatch to the other the path should remain the same, and the path should only change when hyperparameters change.

Reply:
Hum, if the context's path is unique according to the hyperparameters already, then we don't need to add more hashing. Maybe it is me that don't fully understand here and it's probably already okay. 

A solution would therefore be to be sure that the path in `context.get_path()` is unique and according to the hyperparameters of this step and of all the previous. If not, we may need to add `context.get_unique_hp_path()` or something like that.